### DISCUSSION TITLE
adding more attributes to Slot objects

### DISCUSSION THREAD

**Discussion by 0dm** [zauberzeug/nicegui#1388] [Ideas / Feature Requests]
I'm having a small issue with Splitters:
![image](https://github.com/zauberzeug/nicegui/assets/57018940/df2f66b4-d8f0-46f4-ac30-12e824950df3)

I can't seem to get rid of this scroll bar by setting "overflow: hidden;" since there's no style() or class() attribute for the Slot objects (splitter.before & splitter.after). Any workarounds?

```py
 with ui.splitter(value=20).style("flex-wrap: nowrap;") as splitter:
        splitter._props["limits"] = [20, 80]

        # TODO: find a way to set "overflow: hidden;" for the splitter
        with splitter.before:
            ui.table(rows=[meta], columns=meta_col).style("min-width: 50em;")._props[
                "grid"
            ] = True
        with splitter.after:
            global performance_plot_img
            performance_plot_img = ui.interactive_image(
                source=plots[int(ui_dark.value)]
            )
```

   The min-width on the table is needed or else the text will get squished :(

**Comment by falkoschindler** [comment]
Sorry, @0dm, can you create a minimal _reproducible_ code example? As it is right now, I can't try it out which makes it very difficult to debug.

Apart from that: A slot is not an element, but only a place to create one. Therefore it is conceptually impossible to style it. So if you put a table into the `splitter.before` slot, you can style the table, but not the slot.
